app/questionnaire/questionnaire_schema.py

```diff
@@ -503,19 +500,9 @@ def _update_answer_dependencies_for_calculated_summary_dependency(
             block_id=dependent_block["id"]
         )
         for answer_id in calculated_summary_answer_ids:
-            if answer_id in [
-                *self._dynamic_answer_ids,
-                *self._repeating_block_answer_ids,
-            ]:
-                # Type ignore: answer_id is valid so block must exist
-                block_id: str = self.get_block_for_answer_id(answer_id)["id"]  # type: ignore
-                if block_id in self._list_dependent_block_additional_dependencies:
-                    for (
-                        list_block_id
-                    ) in self._list_dependent_block_additional_dependencies[block_id]:
-                        self._answer_dependencies_map[list_block_id].add(
-                            answer_dependent
-                        )
+            if list_name := self.get_list_name_for_answer_id(answer_id):
```

Comment:
Should the name of the method change from `_update_answer_dependencies_for_calculated_summary_dependency` now since we update list dependecies map as well? Should the docstring change as well?

Reply:
Does it make sense to call it `_update_dependencies_for_calculated_summary_dependency`?

Reply:
Yeah good point - updated the docstring too